src/nginx/t/auth_asymmetrickey.t

```diff
@@ -42,22 +42,35 @@ my $NginxPort = ApiManager::pick_port();
 my $BackendPort = ApiManager::pick_port();
 my $ServiceControlPort = ApiManager::pick_port();
 my $PubkeyPort = ApiManager::pick_port();
-my $t = Test::Nginx->new()->has(qw/http proxy/)->plan(22);
+my $t = Test::Nginx->new()->has(qw/http proxy/)->plan(13);
 
 my $config = ApiManager::get_bookstore_service_config;
 $config .= <<"EOF";
 authentication {
-  providers {
-    id: "test_auth"
-    issuer: "test-esp-auth.com"
-    jwks_uri: "http://127.0.0.1:${PubkeyPort}/key"
-  }
+  providers [
+    {
+      id: "test_auth"
+      issuer: "test-esp-auth.com"
```

Comment:
can we have meaning for issuer name?  such as rs256 issuer and es256 issuer?

just want to make new comers life easier to read this test code.

Reply:
Yeah. Totally agreed:)